app/hocs/withProgressPanel.js

```diff
@@ -10,7 +10,8 @@ const mapActionsToProps = (action, props) => ({
   onRetry: () => action.call(props)
 })
 
-export default function withProgressPanel (actions, { title, strategy = alreadyLoadedStrategy, ...options } = {}) {
+export default function withProgressPanel (actions, { title, strategy = recentlyCompletedStrategy, ...options } = {}) {
+  console.log(actions, options, title)
```

Comment:
Sneaky `console.log`

Reply:
Oops. Thought I removed that with a git commit --amend. Somehow they always make it into the PRs :shipit: